src/rtop/reason_utop.ml

```diff
@@ -67,7 +87,8 @@ let init_ocaml () =
   UTop.parse_toplevel_phrase := UTop.parse_toplevel_phrase_default;
   UTop.parse_use_file := UTop.parse_use_file_default;
   UTop.history_file_name :=
-      Some (Filename.concat LTerm_resources.home ".utop-history")
+      Some (Filename.concat LTerm_resources.home ".utop-history");
+  restore_toploop_printing ()
```

Comment:
Is there any reason why `restore_tooploop_printing` needs to be a function? might as well just drop the printer wiring logic in here (like in `init_reason`)